### ISSUE TITLE
NTR: 2-trans,4-cis-decadienoylcarnitine present in blood (Lab: chemistry: carnitine compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8568]
New term request 2-trans,4-cis-decadienoylcarnitine present in blood (segal_050103221121):

**Comment by pnrobinson** [comment]
@MickeySegal
analogous question as [decadienoylcarnitine](https://github.com/obophenotype/human-phenotype-ontology/issues/8567)

**Comment by MickeySegal** [comment]
Acylcarnitine testing by mass spectroscopy is a current clinical assay: https://www.labcorp.com/tests/070228/acylcarnitine-profile-quantitative-plasma

It sounds like the metabolite was picked up in this clinical test.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000478
- label: Elevated circulating 2-trans,4-cis-decadienoylcarnitine concentration
- parents: [Elevated circulating medium-chain acylcarnitine concentration (HP:0035017)](https://hpo.jax.org/app/browse/term/HP:0035017)
- pmid: PMID:2332510|PMID:15344554|PMID:19578400
- def: The concentration of 2-trans,4-cis-decadienoylcarnitine in the blood circulation is above the upper limit of normal.
- comment: 2-trans,4-cis-decadienoylcarnitine is elevated in the blood or plasma of individuals with 2,4-dienoyl-coenzyme a reductase deficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8568
- synonyms:
